drltrace/drltrace_frontend.cpp

```diff
@@ -54,28 +54,41 @@
     exit(1); \
```

Comment:
(Won't let me comment above here, trying to point at line 45)
What about the TESTANY?